test/go-compile-test.el

```diff
@@ -0,0 +1,49 @@
+;;; go-compile-test.el --- unit tests for ‘go-mode’ compilation -*- lexical-binding: t; -*-
+
+;; Copyright 2020 Google LLC
```

Comment:
Is there a reason you used a different copyright assignment and license for this file than normal go-mode.el?

Reply:
I thought this would be a requirement for Googlers to contribute patches, but it's not (https://opensource.google/docs/patching/#license-headers-and-copyright-notices), so I've changed the copyright notice to "The go-mode authors".